testsuite/features/support/env.rb

```diff
@@ -18,9 +18,9 @@
 
 server = ENV['SERVER']
 $debug_mode = true if ENV['DEBUG']
-$long_tests_enabled = true if ENV['LONG_TESTS']
+$long_tests_enabled = true if ENV['LONG_TESTS'] == 'true'
```

Comment:
we should really get rid of that variable and replace it with 2 scopes:
* build images
* virtualization test

Reply:
Out of scope of this PR: this is a bug fix; a refactor is welcome, but please, in a next PR.